### DISCUSSION TITLE
This template created with Bootstrap(5) ?

### DISCUSSION THREAD

**Discussion by askaralimkndr** [tabler/tabler#907] [Q&A] [answered]
Hi, i really like this template, is it designed with used bootstrap 5 framework, if yes, i am not able see the link of bootstrap CSS.
Please help me know about this. Thank you

**Comment by sagegwatkin** [comment, marked as answer]
Hi @askaralimkndr,

Yes this template / UI kit is built on Bootstrap 5.

As for the link to Bootstrap, Tabler compiles Bootstrap styles from the Sass source files. Here's how it works:

1. `npm install`  grabs the Bootstrap source code
2. `npm run start` or `npm run build` triggers a build
2. [package.json](https://github.com/tabler/tabler/blob/dev/package.json#L6) specifies that [gulp](https://gulpjs.com/) is used to build Tabler
3. [gulpfile.js](https://github.com/tabler/tabler/blob/dev/gulpfile.js#L149) specifies that the source Sass files are compiled into the dist CSS files
4. [Bootstrap Sass files](https://github.com/tabler/tabler/blob/dev/src/scss/_bootstrap-components.scss) are compiled together with the Tabler styles to make [tabler.css](https://github.com/tabler/tabler/blob/dev/dist/css/tabler.css), which contains both Tabler and Bootstrap CSS

You can always just copy and paste the pre-compiled [tabler.css](https://github.com/tabler/tabler/blob/dev/dist/css/tabler.min.css) file, this contains both the Tabler styles, and Bootstrap 5 styles, all ready to go.

**Reply by askaralimkndr** [reply]
Thank you for your kind information, i wanted to create web application using this template and want to sell on online market. 
Can i do this ? Please let me know. Thank you

**Reply by sagegwatkin** [reply]
Yes that should be ok. Tabler is using the MIT license which allows for:
-  Commercial use
-  Modification
-  Distribution
-  Private use

You can view the full license information [here](https://github.com/tabler/tabler/blob/dev/LICENSE).

**Reply by askaralimkndr** [reply]
Thank you #sir for kind information. Your information really helps me in my projects with this template.
